deploy/debezium/debezium-connector.yaml

```diff
@@ -27,6 +27,7 @@ objects:
       heartbeat.interval.ms: ${DEBEZIUM_HEARTBEAT_INTERVAL_MS}
       heartbeat.action.query: ${DEBEZIUM_ACTION_QUERY}
       topic.heartbeat.prefix: ${TOPIC_HEARTBEAT_PREFIX}
+      poll.interval.ms: 250
```

Comment:
I feel like we should define this as a parameter like `${DEBEZIUM_HEARTBEAT_INTERVAL_MS}`. Possibly `DEBEZIUM_POLL_INTERVAL_MS`?

Reply:
Good call, lemme do that